src/model/Series.ts

```diff
@@ -588,10 +599,26 @@ class SeriesModel<Opt extends SeriesOption = SeriesOption> extends ComponentMode
             return;
         }
 
-        if (selectedMode === 'multiple') {
+        if (selectedMode === 'series') {
+            const selectedMap = this.option.selectedMap || (this.option.selectedMap = {});
+            const self = this;
+            data.each(idx => {
```

Comment:
I think we should use a special reserved value to indicate all data are selected in this series. For example:

```js
selectedMap = 'all'
```

It will be much faster and simpler than traverse all data. Also it has benefits that when new data is added. It's selected automatically. However current implentation will not include this new data.

Then we can check if selectedMap is 'all' in `isSelected`. Also `_selectedDataIndicesMap` is not necessary in this case. `getSelectedDataIndices ` can return all indices of data.

Reply:
But if some data have specify `select.disable = false`? Just like the test case I have added. @pissang

Reply:
It doesn't matter, disabled can be checked in the isSelected method

Reply:
Add `isSelectSeries` @pissang

Reply:
selected states should be saved in the option because it's is the single source of truth. Why not using selectedMap: 'all'?

Reply:
Right, using `selectedMap` now @pissang